db/migrate/20151018090641_create_persistent_volume_claim.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePersistentVolumeClaim < ActiveRecord::Migration
+  def change
+    create_table :persistent_volume_claims do |t|
+      t.belongs_to :ems, :type => :bigint
+      t.belongs_to :persistent_volume, :type => :bigint
+      t.string     :ems_ref
+      t.string     :name
+      t.timestamp  :ems_created_on
+      t.timestamp  :created_on
+      t.string     :resource_version
+      t.text       :desired_access_modes, :array => true, :default => []
+      t.string     :volume_name
+      t.string     :phase
+      t.text       :actual_access_modes, :array => true, :default => []
+      t.text       :capacity
```

Comment:
@blomquisg ```volume_name``` and ```capacity``` are indeed the same, and actually also ```actual_access_modes```. They all represent data regarding the persistent volume backing the claim, so we may remove them as you suggested. On the other hand, ```phase``` represents the current phase of the claim itself which is not necessary depend on the volume (if not bounded for example). @simon3z can you please confirm that?